### ISSUE TITLE
Detect proper method casing

### ISSUE THREAD

**Issue by OndraM** [lmc-eu/php-coding-standard#14]
Codestyle should detect when wrong casing (which differs from parent class/interface) is used.
https://github.com/lmc-eu/matej-client-php/pull/72/files/81f858f6395a328912b2d46e9adfb0c02641ef10#diff-ad0bc5361b920e3a3e22dce7973789a4R18

```php
class MyTest extends PHPUnit\Framework\TestCase
    protected function setup(): void { // should be setUp
    ...
    }
}
``` 

I've seem some check for this, not sure where.
